x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvSliceTests.java

```diff
@@ -306,7 +306,16 @@ private static void bytesRefs(List<TestCaseSupplier> suppliers) {
         }));
 
         suppliers.add(new TestCaseSupplier(List.of(DataType.GEO_SHAPE, DataType.INTEGER, DataType.INTEGER), () -> {
-            List<Object> field = randomList(1, 5, () -> new BytesRef(GEO.asWkt(GeometryTestUtils.randomGeometry(randomBoolean()))));
+            var pointCounter = new MvAppendTests.GeometryPointCountVisitor();
+            List<Object> field = randomList(
+                1,
+                5,
```

Comment:
Maybe reduce it to 3? I noticed that MvAppendTests set it to 3.

Reply:
5 should be fine as well, I think the biggest problem was the size of the single shape.